languages.toml

```diff
@@ -3680,6 +3680,17 @@ language-servers = [
 indent = { tab-width = 2, unit = "  " }
 grammar = "javascript"
 
+[[language]]
+name = "gherkin"
+scope = "source.feature"
+file-types = ["feature"]
+comment-token = "#"
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "gherkin"
+source = { git = "https://github.com/SamyAB/tree-sitter-gherkin", rev = "43873ee8de16476635b48d52c46f5b6407cb5c09" }
+
```

Comment:
ok actually one last thing 😅

Could you move this block above `gjs` or below `gts`? Where it is now it interrupts `gjs`'s block and accidentally takes its `auto-pairs` config

Reply:
I did not see that it was in the middle of a block, my bad :sweat_smile:

It's now bellow `gts`